src/main/java/ChatBot.java

```diff
@@ -6,9 +6,14 @@
 
 public class ChatBot {
     private boolean isRunning;
+    private String[] responses;
+    private static final int ResponseLimit = 100;
```

Comment:
Perhaps the variable name for the constant can be changed to `RESPONSE_LIMIT` to adhere to Java coding standards!